pkg/integration/tests/patch_building/copy_patch_to_clipboard.go

```diff
@@ -0,0 +1,53 @@
+package patch_building
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+var BuildPatchAndCopyToClipboard = NewIntegrationTest(NewIntegrationTestArgs{
+	Description:  "Create a patch from the commits and copy the patch to clipbaord.",
+	ExtraCmdArgs: "",
+	Skip:         false,
+	SetupConfig:  func(config *config.AppConfig) {},
+	SetupRepo: func(shell *Shell) {
+		shell.NewBranch("branch-a")
+		shell.CreateFileAndAdd("file1", "first line\n")
+		shell.Commit("first commit")
+
+		shell.NewBranch("branch-b")
+		shell.UpdateFileAndAdd("file1", "first line\nsecond line\n")
+		shell.Commit("update")
+
+		shell.Checkout("branch-a")
+	},
+	Run: func(t *TestDriver, keys config.KeybindingConfig) {
+		t.Views().Branches().
+			Focus().
+			Lines(
+				Contains("branch-a").IsSelected(),
+				Contains("branch-b"),
+			).
+			Press(keys.Universal.NextItem).
+			PressEnter().
+			PressEnter()
+		t.Views().
+			CommitFiles().
+			Lines(
+				Contains("M file1").IsSelected(),
+			).
+			PressPrimaryAction()
+
+		t.Views().Information().Content(Contains("building patch"))
```

Comment:
I've pushed a commit with some changes. The first one is here: I was getting issues with the primary action button being pressed and then the menu being opened before the patch had been started. To fix this we're now asserting that we're in the patch building mode via the information view.